Library/Homebrew/cmd/deps.rb

```diff
@@ -211,4 +229,61 @@ def recursive_deps_tree(f, prefix, recursive)
 
     @dep_stack.pop
   end
+
+  class Dependent
```

Comment:
How about calling this `CaskDependent` and putting it in its own class somewhere?